### ISSUE TITLE
Trains have jumpy/jittery movement

### ISSUE THREAD

**Issue by ZehMatt** [OpenLoco/OpenLoco#2059]
**Version information:**
 - Platform: *
 - Version: 23.07
 - Build/commit: [e.g. f08479f]

**Describe the bug**
It seems like any vehicles that aren't the head have some jumpy jittery movement.
Here is a video with a time scale of 0.1 and tween ing/uncap fps off:
https://github.com/OpenLoco/OpenLoco/assets/5415177/6374ca1f-0399-4a6c-920a-0da6683456e1

Here is a video with tweening on, where its actually getting worse then:
https://github.com/OpenLoco/OpenLoco/assets/5415177/a809a319-4ad5-4b77-a406-790d5a5419e3

**Comment by NSTrains** [comment]
That always happens after 40 cars

**Comment by AaronVanGeffen** [comment]
You're right that there seems to be some unintentional jitteriness to the head of some longer vehicles. I think we should solve that when possible. However, I do like the aesthetic of the trains hopping vertically -- that seems intentional and should be preserved, imo.

**Comment by ZehMatt** [comment]
> You're right that there seems to be some unintentional jitteriness to the head of some longer vehicles. I think we should solve that when possible. However, I do like the aesthetic of the trains hopping vertically -- that seems intentional and should be preserved, imo.

Seems a bit unrealistic to me unless there is a constant speed change, I could look away for the vertical hops but the horizontal jumps are a bit akward, track trains are typically quite even and would affect all all of them.
